tests/users/test_api_user.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from flask import json
+from flask_restplus import marshal
+
+from app.api.models.user import public_user_api_model
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1, user2
+from tests.test_utils import get_test_request_header
+
+
+class TestUserApi(BaseTestCase):
```

Comment:
```suggestion
class TestnGetOtherUserApi(BaseTestCase):
```
To be more explicit about the tests that you are running here, which is about get other user profile API